### DISCUSSION TITLE
[项目]task to earn

### DISCUSSION THREAD

**Discussion by uvd** [hoh-zone/SuiStartrek#96] [Ideas]
[项目]task to earn
https://github.com/movefuns/detask
需求 bounty 系统 参考 https://dorahacks.io/zh/daobounty
degrant系统 https://dorahacks.io/zh/dora
集体task系统，完成学习认证获取token

链接sui钱包 github登陆绑定等等

**Comment by Euraxluo** [comment]
@Euraxluo ACK

**Comment by Wulabalabo** [comment]
@Wulabalabo 非常有意思的想法，我来做一版产品MVP
